jwst/extract_1d/extract.py

```diff
@@ -686,23 +680,20 @@ def aperture_from_wcs(wcs: WCS, verbose: bool) -> Union[NamedTuple, None]:
         bounding_box = wcs.bounding_box
         got_bounding_box = True
     except AttributeError:
-        if verbose:
-            log.info("wcs.bounding_box not found; using wcs.domain instead.")
+        log.info("wcs.bounding_box not found; using wcs.domain instead.")
```

Comment:
Because the original code seemed to be designed to emit this message only if the obsolete "verbose" param was set, perhaps the log type should be changed to `log.debug()`, so that it only appears when the standard debug/verbose command-line param is set.

Reply:
Agreed on all of these; I thought I had done this to them all, but I guess I was hasty in my mass deletion event.